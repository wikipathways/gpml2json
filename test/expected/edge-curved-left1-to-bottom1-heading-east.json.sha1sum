b6364e9fbea7244460fa906410ba0885a77ebaf7 ?-
