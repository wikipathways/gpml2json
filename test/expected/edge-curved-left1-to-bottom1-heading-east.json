{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["e4428","b4ae1","e795e"],"drawAs":"rect","gpmlElementName":"Pathway","height":818.1126371280204,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":1047.7865523249097,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/3c138d3b2eaf975e771764919ee927175f08446182325bfa6825c0f541d8a42d"},"entityMap":{"b4ae1":{"id":"b4ae1","textContent":"GeneProduct","wpType":"GeneProduct","x":138.9851404400389,"y":88.5516034023422,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32978,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"e795e":{"id":"e795e","textContent":"0010","wpType":"GeneProduct","x":16.726305049501832,"y":58.393505932942674,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32979,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"e4428":{"id":"e4428","drawAs":"CurvedLine","borderWidth":1,"zIndex":32977,"points":[{"isAttachedTo":"e795e","attachmentDisplay":{"position":[1,0.5],"offset":[0,0]},"x":96.72630504950183,"y":68.39350593294267,"orientation":[1,0]},{"x":116.72630504950183,"y":98.47255466764244},{"y":128.5516034023422,"x":147.85572274477036},{"isAttachedTo":"b4ae1","attachmentDisplay":{"position":[0.5,1],"offset":[0,0]},"x":178.9851404400389,"y":108.5516034023422,"orientation":[0,-1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["e795e","b4ae1"]}}}
